js/mock/cars.js

```diff
@@ -0,0 +1,363 @@
+                id: '07',
+                brand: 'Volkswagen ',
+                model: 'Polo',
+                color: 'Silver-grey',
+                year: 2003,
+                country: 'German',
+                price: 5600,
+                power: 'gasoline',
+                image: 'https://cdn1.riastatic.com/photosnew/auto/photo/volkswagen_polo__467907701f.webp',
+                video: 'https://www.youtube.com/watch?v=WyawJQPxQ4M'
+                },
+                {
+                  id: '08',
+                  brand: 'Toyota',
+                  model: 'Camry',
+                  color: 'brown metalic',
+                  year: 2018,
+                  country: 'Japan',
+                  price: 26300,
+                  power: 'hybrid',
+                  image: 'https://cdn0.riastatic.com/photosnew/auto/photo/toyota_camry__474110310f.webp',
+                  video: 'https://www.youtube.com/watch?v=UKBjTSgYNSU'
+                  },
+                  {
+                    id: '09',
+                    brand: 'Volkswagen',
+                    model: 'Transporter t4',
+                    color: 'black',
+                    year: 1997,
+                    country: 'German',
+                    price: [3600,'special deal'],
+                    power: 'gasoline',
+                    image: 'https://cdn2.riastatic.com/photosnew/auto/photo/volkswagen_t4-transporter-pass__471754627f.webp',
+                    video: 'https://www.youtube.com/watch?v=6tpsHrxBuPg'
+                    },
+                    {
+                      id: '10',
+                      brand: 'Nissan',
+                      model: 'Teana',
+                      color: 'black',
+                      year: 2007,
+                      country: 'Japan',
+                      price: 5600,
+                      power: 'gasoline',
+                      image: 'https://cdn2.riastatic.com/photosnew/auto/photo/nissan_teana__470335327f.webp',
+                      video: 'https://www.youtube.com/watch?v=lcJ1C-3efHQ'
+                      },
+                      {
+                        id: '11',
+                        brand: 'Chery ',
+                        model: 'Tiggo',
+                        color: 'metalic',
+                        year: 2022,
+                        country: 'China',
+                        price: 13922,
+                        power: 'gasoline',
+                        image: 'https://cdn.riastatic.com/photosnewr/auto/new_auto_storage/chery-tiggo-2-pro__1565941-620x465x72.webp',
+                        video: 'https://www.youtube.com/watch?v=AcZxzFOgp4o'
+                       },
+                       {
+                          id: '12',
+                          brand: 'Opel ',
+                          model: 'Crossland',
+                          color: 'white',
+                          year: 2022,
+                          country: 'German',
+                          price: 16781 ,
```

Comment:
Please remove extra space before the coma. Please check all of the objects below.